app/templates/components/modals/proposal-delete-modal.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="header">
+  {{t 'Are you sure you would like to delete this proposal?'}}
+  <div class="muted small text">
+    {{t 'Deleting the proposal will delete all the data associated with it.'}}
+  </div>
+</div>
+<div class="content">
+  <form class="ui {{if isLoading 'loading'}} form" id="delete-form" autocomplete="off" {{action deleteProposal on='submit' preventDefault=true}}>
+    <div class="field">
+      <div class="label">
+        {{t 'Proposal name you are deleting:'}}
+      </div>
+      {{input type='text' name='confirm_name' value=proposalName disabled=true}}
```

Comment:
umm.. why is the input disabled ?

Reply:
I thought of not implementing it the way event is deleted as Proposal title can be long. Should I add that confirmation modal, to enter the name of proposal before deletion?

Reply:
hmm that's a valid point , but @pradeepgangwar then if you are not implementing it like that .. what's the point in displaying the proposal name in a  text field?  Just display it as a slightly differently styled text instead of a text field. In a single sentence, 
`Are you sure you would like to proceed forward with withdrawing the session titled _sessionname_ `?
And user can select yes or no.

Reply:
@CosmicCoder96 Ok I will fix this. :)